libid/fractals.cpp

```diff
@@ -728,3 +881,29 @@ int other_julia_fp_per_pixel()
     }
     return 0;
 }
+
+int burning_ship_fp_fractal()
+{
+    DComplex z, q;
```

Comment:
One variable per declaration and move declarations as far down as they can go to where they are first used/initialized.